src/emcpy/plots/create_map.py

```diff
@@ -16,7 +16,7 @@ class CreateMap:
 
     def __init__(self, fig=plt.figure(figsize=(12, 8)),
                  domain=Domain('global'),
-                 projection=MapProjection('plcarr')):
+                 proj_obj=MapProjection('plcarr')):
```

Comment:
I think I prefer something like `mapproj` over `proj_obj` but I won't request it being changed unless @aerorahul feels strongly about some different name than `proj_obj`

Reply:
anything different that `projection`, since that is the root of the confusion.